src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotHardCodeEncryptionKeyTests.cs

```diff
@@ -272,7 +273,9 @@ public void CreateEncryptor(byte[] rgbKey, byte[] rgbIV)
         [Fact]
         public void Test_PassTaintedSourceInfoAsParameter_SinkProperties_Interprocedual_Diagnostic()
         {
-            VerifyCSharpWithDependencies(@"
+            Parallel.For(1, 100, _ =>
```

Comment:
I can revert the unit test changes if we think that is more appropriate, especially given the newly added asserts will fire if the underlying issue crops up again. Though having it does help us ensure we have a unit test validating our flow analysis framework under large number of simultaneous compilations.